modules/rest-api/pages/rbac.adoc

```diff
@@ -1,4 +1,5 @@
 = Role Based Admin Access (RBAC)
+:page-aliases: rest-api:rest-bucket-auth
```

Comment:
```suggestion
:page-aliases: rest-bucket-auth
```
Since the page alias resides in the same module, you shouldn't have to specify the module again.